docs/persistence/bigquery.md

```diff
@@ -1,9 +1,9 @@
-# Bigquery Datasets
+# Google Cloud BigQuery Dataset
 
-!!! info
-    This feature is only available in [GCP](../../clusters/gcp) clusters.
```

Comment:
Er det ikke forstatt lurt å tydeliggjøre at det kun er tilgjengelig i GCP? Gjelder egentlig flere sider også isåfall

Reply:
Kanskje vi skal snu på det og si at det ikke er tilgjengelig on-premises i stedet? Så er vi "cloud-first" 😛

Reply:
Jeg føler dette er litt implicit ettersom det er Google Cloud Bigquery og under `spec.gcp` ble bare veldig mange sånne bokser på sidene.

Reply:
Ble tydeligere nå med "Google Cloud" som prefiks i tittel i hvert fall, så da tenker jeg at det er greit 👍